zerver/lib/actions.py

```diff
@@ -1839,26 +1839,35 @@ def already_sent_mirrored_message_id(message: Message) -> Optional[int]:
         return messages[0].id
     return None
 
-def extract_recipients(s: Union[str, Iterable[str]]) -> List[str]:
+def extract_recipients(s: Union[Union[str, int], Iterable[Union[str, int]]]) -> List[str]:
```

Comment:
I think we only need to support the `Iterable[int]` case.  (I.e. we can require the `[1, 2, 3]` style syntax).

Reply:
@timabbott: I'm not sure what you mean, wouldn't we still need `Iterable[str]` for emails? Or are you referring to `extract_recipients_by_id`?

Reply:
I meant the `Union[str, int]` earlier in the list, for supporting just an integer (not inside a list).